app/src/test/java/com/puzzlebench/clean_marvel_kotlin/presentation/mvp/CharacterPresenterTest.kt

```diff
@@ -44,57 +50,134 @@ class CharacterPresenterTest {
                 getCharacterRepositoryUseCase,
                 saveCharacterRepositoryUseCase,
                 subscriptions)
-
-
     }
 
-    @Ignore
+    @Test
     fun init() {
         val itemsCharacters = listOf(1..5).map {
             mock(Character::class.java)
         }
         val observable = Single.just(itemsCharacters)
-        Mockito.`when`(getCharacterServiceUseCase.invoke()).thenReturn(observable)
-        Mockito.`when`(getCharacterRepositoryUseCase.invoke()).thenReturn(emptyList())
+        `when`(getCharacterServiceUseCase.invoke()).thenReturn(observable)
+        `when`(getCharacterRepositoryUseCase.invoke()).thenReturn(emptyList())
         characterPresenter.init()
         verify(view).init()
         verify(characterServiceImp).getCharacters()
         verify(characterRepository).getAll()
         verify(view).hideLoading()
+        verify(view).showFAB()
         verify(view).showCharacters(itemsCharacters)
     }
 
-    @Ignore
+    @Test
     fun reposeWithError() {
-        Mockito.`when`(getCharacterServiceUseCase.invoke()).thenReturn(Single.error(Exception("")))
+        `when`(getCharacterServiceUseCase.invoke()).thenReturn(Single.error(Exception("")))
```

Comment:
you can use EMPTY_STRING instead of "", but a better approach is to have a better error description message to differentiate from a possible default state ("")

Reply:
done